aten/src/ATen/native/native_functions.yaml

```diff
@@ -5372,6 +5372,11 @@
 
 - func: meshgrid(Tensor[] tensors) -> Tensor[]
 
+# TODO: Two weeks after this lands, combine these two overloads,
+#       making "indexing" optional. These are temporarily distinct for
+#       forward-compatibility reasons.
+- func: meshgrid.indexing(Tensor[] tensors, *, str? indexing) -> Tensor[]
```

Comment:
Which of the two above signatures gets matched when calling `meshgrid(t)`?

Does that matching rely on the way PyTorch's dispatch system orders the signatures?

Reply:
No, I think it 100% relies on the torch.functional() implementation. Does this impact the FC strategy? Do I need to change torch.functional() to continue delegating to the legacy implementation for the two week FC period?

Reply:
I think so. See comment below

Reply:
resolving this, let's take the discussion to the other comment thread.